gamemode/load_modules.lua

```diff
@@ -0,0 +1,24 @@
+local moduleDir = GM.FolderName .. "/gamemode/modules/"
+
+local _, modules = file.Find( moduleDir .. "*", "LUA" )
+
+for _, moduleName in pairs( modules ) do
+    local path = moduleDir .. moduleName .. "/"
```

Comment:
Could be nice to add some prints in here, so it's clear what modules have been loaded successfully.
Something like
`print( "[RVR] Loading module " .. moduleName )`
and
`print( "[RVR] Finished loading modules" )`
at the end.

Reply:
Could even put some information about which realms have been used in the print, like "Loaded shared module" if server and client side files were found!